src/compas/files/urdf.py

```diff
@@ -322,7 +322,7 @@ def build_kwargs_by_type(cls, elements, parser_type, default_namespace):
         return result
 
 
-class URDFGenericElement(Base):
+class URDFGenericElement(Data):
```

Comment:
In this class (as in many others in the robot model implementation), the methods `to_json`/`from_json` could be removed now.